components/nodemanager-service/pgdb/node-managers-cloud.go

```diff
@@ -229,17 +229,24 @@ func (db *DB) AddManagerNodesToDB(nodesArr []*manager.ManagerNode, managerId str
 			}
 		}
 
+		var dbNodeID string
 		err = Transact(db, func(tx *DBTrans) error {
-			uuid, err = tx.SelectStr(sqlInsertManagerNode, uuid, mgrType, item.Id, jsonTc, item.Name, item.Region, managerAcctId)
+			logrus.Debugf("adding manager node %s with cloud details: %s %s %s ", item.Name, item.Id, item.Region, managerAcctId)
```

Comment:
I'm cool with these kinda debugging log statements,  but I'd prefer something like:

```suggestion
			logctx := logrus.WithFields(logrus.Fields{
				"name": item.Name,
				"id": item.Id,
				"region": item.Region,
				"account_id": managerAcctId,
			})
			logctx.Debug("adding manager node")
```

so then we can reuse logctx at line 242 to ensure we get consistent logging.

Reply:
oooh nice!! ya ill change it to that!